README.md

```diff
@@ -1,113 +1,283 @@
-# Classes - Properties
+# Classes & Properties
 
 ![NameOfPerson](http://i.imgur.com/GAIzIs0.jpg?1)  
 
 > People often say that motivation doesn't last. Well, neither does bathing--that's why we recommend it daily. -[Zig Ziglar](https://en.wikipedia.org/wiki/Zig_Ziglar)
- 
 
 ## Learning Objectives
 
-* 
+* Understand what properties are in Swift classes
+* Create your own stored properties
+* Create computed properties that are based on the values of other properties
+* Understand the difference between constant and variable properties
+* Understand the difference between constant and variable objects
 
+## Properties
 
+Think about a car for a second. A car is a fairly generic object: It's a thing you drive that has four wheels, a body, an engine, and some seats. (It has a lot of other stuff, too, like a transmission and brakes and stuff, but let's not get too hung up on details.) What, then, differentiates one car for another? When you're looking for your car in a parking lot, how can you pick out your car from the dozens or hundreds of others parked there, too?
 
-## Outline / Notes
+Cars, like all things in real life, have certain attributes that let you tell the difference between them. Your car, for example, might be a green Toyota with four doors and a black interior. It also has other identifying items, like a license plate, that let you easily pick out _your_ car from another one. You wouldn't mistake a bright red two-door Lamborghini for your green Toyota!
 
-*  I've included an example below I provided to the in person immersive class here (might be a little too much in that I used `reduce` here).. but I think you get the point.
+In Swift, the attributes that separate an instance of a class from other instances are called _properties_. Properties are what drive instances of classes: If all instances were the same, you wouldn't need properties, but because objects have properties, you can create several instances of the same type (or class) that have different qualities.
 
-* I like the example Apple users in their documentation book.. but feel free to run wild. I want them to be able to understand the difference between stored properties and computed properties along with computed read-only properties.
+There are several types of properties in Swift. Let's look at the first, simplest type of property: _stored properties_.
 
```

Comment:
Great intro. I'm stealing this when I teach this in person. Much better than unicorns and birds.